src/conn/conn_log.c

```diff
@@ -404,7 +404,7 @@ __log_remove_once(WT_SESSION_IMPL *session, uint32_t backup_file)
     WT_ORDERED_READ(dbg_val, conn->debug_ckpt_cnt);
     if (FLD_ISSET(conn->debug_flags, WT_CONN_DEBUG_CKPT_RETAIN) && dbg_val != 0)
         min_lognum = WT_MIN(conn->debug_ckpt[dbg_val - 1].l.file, min_lognum);
-    WT_ORDERED_READ(dbg_val, conn->debug_log_cnt);
+    WT_READ_ONCE(dbg_val, conn->debug_log_cnt);
```

Comment:
I still think we should do this in your debug_blah ticket.

Reply:
After offline discussion we'll remove this line and move the PR to draft mode. We can merge the PR when a future ticket has need for this macro